cicd/main.yml

```diff
@@ -0,0 +1,118 @@
+#trigger:
+#  batch: true
+#  branches:
+#    include:
+#      - refs/heads/master
+#  paths:
+#    exclude:
+#    - README.md
+
+pool: Computer-Vision-Team-Amsterdam
+
+parameters:
+  - name: AMLModelVersion
+    displayName: Version of the OOR model in AML
+    type: string
+    default: 1
+
+resources:
+  repositories:
+    - repository: self
+
+variables:
+  - template: variables.yaml
+
+stages:
+  - stage: Prerequisites
+    displayName: "Collect all prerequisites"
+    jobs:
+      - template: "prerequisites-pipeline.yml"
```

Comment:
I find it a bit hard to follow how they all come together, maybe because in this case prerequisites-pipeline.yml is a template but it is not under the template folder, while download-aml-model is also a template, but it is under the template folder. 

I still find it logical to have `prebuild-pipeline,` `build-pipeline`, `main` and `deploy` under `cicd`, but move `prerequisites-pipeline` under templates folder.

Reply:
Moved!